Makefile

```diff
@@ -3,16 +3,16 @@ BACKEND := bookclub-voting-backend
 FRONTEND := bookclub-voting-frontend
 
 up:
-	docker-compose up --build
+	docker compose up --build
```

Comment:
So apparently this way doesn't work for me because I didn't set up the CLI plugin :smiling_face_with_tear: 
Are you gonna make me set that up or do you  mind changing this back?

Reply:
I'll do it myself